compiler/il/OMROpcodes.enum

```diff
@@ -6556,7 +6556,7 @@ OPCODE_MACRO(\
    /* .properties4          = */ 0, \
    /* .dataType             = */ TR::VectorDouble, \
    /* .typeProperties       = */ ILTypeProp::Size_16 | ILTypeProp::Floating_Point | ILTypeProp::Vector, \
-   /* .childProperties      = */ TWO_SAME_CHILD(ILChildProp::UnspecifiedChildType), \
+   /* .childProperties      = */ ONE_CHILD(ILChildProp::UnspecifiedChildType), \
```

Comment:
The only other implementation is on PPC, and its implementation uses one child. It must be a mistake because I can't think of a reason a unary operator would require two children. I can't find any uses of the opcode in omr or openj9, which is probably why this wasn't discovered earlier.